documentation/Building_Bundles.md

```diff
@@ -40,8 +40,8 @@ to the standard OSGi Bundle headers.
     This optional header can be used to indicate which packages of your Bundle need to be scanned for Spring beans.  
```

Comment:
Please document that the packages in this header are scanned recursively, and that they MUST NOT be the parent package of any item listed in Import-Package